### ISSUE TITLE
Error when opening Cloud Code Terminal in Win10: The token '&&' is not a valid statement separator in this version

### ISSUE THREAD

**Issue by rnwolf** [GoogleCloudPlatform/cloud-code-intellij#3017]
### Expected Behavior

Open Cloud Code Terminal with with path set and able to run SDK tools

### Actual Behavior

![image](https://user-images.githubusercontent.com/117415/139031560-b34ece18-bb4d-4b7b-9f8d-7a7e0164cd8e.png)

```
set PATH="C:\Users\rnwol\AppData\Local\cloud-code\bin\versions\0991fad040597caf6507a2978b6c8aefecf577549a14fdeabf5050cc920bc043";"C:\Users\rnwol\AppData\Lo
cal\google\ct4j-cloud-sdk\LATEST\google-cloud-sdk\bin";%PATH% && cls && echo "Dependencies configured in 'Settings -> Cloud Code' (Cloud SDK/gcloud, skaffold, kubectl, and minikube) have been added t
o the PATH for this terminal session."
At line:1 char:218
+ ... \google\ct4j-cloud-sdk\LATEST\google-cloud-sdk\bin";%PATH% && cls &&  ...
+                                                                ~~
The token '&&' is not a valid statement separator in this version.
At line:1 char:225
+ ... \ct4j-cloud-sdk\LATEST\google-cloud-sdk\bin";%PATH% && cls && echo "D ...
+                                                 

The token '&&' is not a valid statement separator in this version.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : InvalidEndOfLine

```

### Additional Information

- IDE type  Pycharm
- IDE version: 2021.2.3
- Cloud Code version (Settings > Plugins > Cloud Code): 21.10.1-IB.1
- Cloud SDK (Settings > Cloud Code > Cloud SDK)
  - Are you allowing the plugin to manage the Cloud SDK:  yes
  - Version of the Cloud SDK: 362.0.0
- Operating System: Win 10 Pro 21H1 OS Build 19043

**Comment by emmanuelbaah** [comment]
Taking a look during planning to see if can be added to next sprint

**Comment by ruomengz** [comment]
This should be fixed in our recent releases from [v22.3.1](https://github.com/GoogleCloudPlatform/cloud-code-intellij/releases/tag/v22.3.1).

Please update the plugin version and feel free to let us know if there's still issue. Thanks!
